cheatsheets/ssrf.md

```diff
@@ -1,5 +1,9 @@
 ## SSRF
 
+**How to SSRF by Jobert Abma
```

Comment:
- Please use the original title here: _"How To: Server-Side Request Forgery (SSRF)"_;
- It appears that you wanted to make this title bold, but have placed the `**` at the beginning by accident.